김민기/4월 2주차/boj0096_210408_2589_g5/백준_2589번_보물섬.java

```diff
@@ -0,0 +1,79 @@
+package boj.boj0096_210408_2589_g5;
+
+import java.util.*;
+import java.io.*;
+
+public class 백준_2589번_보물섬 {
+	private static class Pos{
+		int row;
+		int col;
+		public Pos(int row, int col){
+			this.row = row;
+			this.col = col;
+		}
+	}
+	private static final BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+	private static StringTokenizer st;
+
+	private static final int[][] DIR = {{-1, 1, 0, 0}, {0, 0, -1, 1}};
+	private static int N;
+	private static int M;
+
+	private static boolean[][] map;
+	private static boolean[][] visited;
+
+	private static void init() throws IOException{
+		st = new StringTokenizer(br.readLine());
+		N = Integer.parseInt(st.nextToken());
+		M = Integer.parseInt(st.nextToken());
+
+		map = new boolean[N][M];
+
+		for (int i = 0; i < N ; i++) {
+			char[] in = br.readLine().toCharArray();
+			for (int j = 0; j < M; j++) {
+				if(in[j] == 'L')    map[i][j] = true;
```

Comment:
in 배열을 그대로 사용하지 않고 map 배열로 바꾸신 이유가 궁금합니다!

Reply:
@Su-Kyung 
값이 `L`과 `W` 밖에 없어서 `boolean`으로 사용해도 될 것 같아서 바꿔 받았습니다 !

```suggestion
				map[i][j] = in[j] == 'L' ;
```